cmd/gotypeconv/main_test.go

```diff
@@ -2,9 +2,55 @@ package main
 
 import (
 	"bytes"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
 	"testing"
+
+	ddiff "github.com/kylelemons/godebug/diff"
```

Comment:
**[staticcheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
could not import github.com/kylelemons/godebug/diff (cannot find package "github.com/kylelemons/godebug/diff" in any of: